arangod/Cluster/ClusterMethods.cpp

```diff
@@ -3960,70 +3959,72 @@ arangodb::Result removeLocalBackups(
-        commError, std::string("Communication error while deleting backup")
-        + backupId + " on " + req.destination);
+      return arangodb::Result(commError,
+                              std::string(
+                                  "Communication error while deleting backup") +
+                                  backupId + " on " + req.destination);
     }
     TRI_ASSERT(res.answer != nullptr);
     auto resBody = res.answer->toVelocyPackBuilderPtrNoUniquenessChecks();
     VPackSlice resSlice = resBody->slice();
     if (!resSlice.isObject()) {
       // Response has invalid format
-      return arangodb::Result(
-        TRI_ERROR_HTTP_CORRUPTED_JSON,
-        std::string("failed to remove backup from ") + req.destination + ", result not an object");
+      return arangodb::Result(TRI_ERROR_HTTP_CORRUPTED_JSON,
+                              std::string("failed to remove backup from ") +
+                                  req.destination + ", result not an object");
     }
 
     if (!resSlice.hasKey("error") || !resSlice.get("error").isBoolean() ||
         resSlice.get("error").getBoolean()) {
-
       int64_t errorNum = resSlice.get("errorNum").getNumber<int64_t>();
 
       if (errorNum == TRI_ERROR_FILE_NOT_FOUND) {
         notFoundCount += 1;
         continue;
       }
 
-      std::string errorMsg = std::string("failed to delete backup ") + backupId + " on " + req.destination
-        + ":" + resSlice.get("errorMessage").copyString() + " (" + std::to_string(errorNum) + ")";
+      std::string errorMsg = std::string("failed to delete backup ") +
+                             backupId + " on " + req.destination + ":" +
+                             resSlice.get("errorMessage").copyString() + " (" +
+                             std::to_string(errorNum) + ")";
 
       LOG_TOPIC("9b94f", ERR, Logger::BACKUP) << errorMsg;
       return arangodb::Result(static_cast<int>(errorNum), errorMsg);
     }
   }
 
-  LOG_TOPIC("1b318", DEBUG, Logger::BACKUP) << "removeLocalBackups: notFoundCount = " << notFoundCount << " " << dbServers.size();
+  LOG_TOPIC("1b318", DEBUG, Logger::BACKUP)
+      << "removeLocalBackups: notFoundCount = " << notFoundCount << " "
+      << dbServers.size();
 
   if (notFoundCount == dbServers.size()) {
-    return arangodb::Result(TRI_ERROR_HTTP_NOT_FOUND, "Backup " + backupId + " not found.");
+    return arangodb::Result(TRI_ERROR_HTTP_NOT_FOUND,
+                            "Backup " + backupId + " not found.");
   }
 
   deleted.emplace_back(backupId);
   LOG_TOPIC("04e97", DEBUG, Logger::BACKUP) << "Have located and deleted " << backupId;
 
   return arangodb::Result();
-
 }
 
-std::vector<std::string> const versionPath = std::vector<std::string>{"arango", "Plan", "Version"};
+std::vector<std::string> const versionPath =
+    std::vector<std::string>{"arango", "Plan", "Version"};
 
 arangodb::Result hotBackupCoordinator(VPackSlice const payload, VPackBuilder& report) {
-
   // ToDo: mode
```

Comment:
Is this an open TODO? 
Or a leftover or something planned for the future?

Reply:
I am not aware of anything TODO here. I think we can remove the TODO here.